Breezy-Traveler/UtilityExtensions+Interfacable.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  UtilityExtensions+Interfacable.swift
+//  Breezy-Traveler
+//
+//  Created by Erick Sanchez on 12/13/18.
+//  Copyright © 2018 Phyllis Wong. All rights reserved.
+//
+
+import UIKit
+
+protocol InstantiatableFromXib {
+    static func instantiateFromXib() -> Self
+}
+
+extension InstantiatableFromXib where Self: UIViewController {
+    static func instantiateFromXib() -> Self {
+        return self.init(nibName: String(describing: self), bundle: nil)
+    }
+}
+
+protocol RegisterableCell {
+    static var identifier: String { get }
+    static var nib: UINib { get }
+}
```

Comment:
This will help register table view cells much easier: example here

```swift
    override func viewDidLoad() {
        super.viewDidLoad()

        tripsTableView.register(TripsTVCell.self)
    }
```